### DISCUSSION TITLE
[v3] search feature not working on the official home page.

### DISCUSSION THREAD

**Discussion by sajjadhossainshohag** [livewire/livewire#6221] [4. General]
Getting error on console. 

**Screenshot below:**
![Screenshot 2023-08-06 095110](https://github.com/livewire/livewire/assets/63788037/43966bd1-b40b-4c10-adc6-536110335bfb)

**Comment by joshhanley** [comment]
@sajjadhossainshohagbd thanks! We will have a look into it.

**Comment by jasonlbeggs** [comment]
@sajjadhossainshohagbd this should be fixed now. Thanks for the heads up!
